src/allmydata/scripts/magic_folder_cli.py

```diff
@@ -1,7 +1,13 @@
 
 import os
+import urllib
+from sys import stderr
 from types import NoneType
 from cStringIO import StringIO
+from datetime import datetime
+
+import humanize
+import simplejson  # XXX why not built-in json lib?
```

Comment:
There was an intentional decision to use simplejson; I can't remember the details or find the ticket right now.